FV3/io/FV3GFS_io.F90

```diff
@@ -3008,7 +2993,8 @@ subroutine send_diag_manager_controlled_diagnostic_data(Time, Diag, Atm_block, I
 
     do index = 1, DIAG_SIZE
       if (trim(Diag(index)%name) .eq. '') exit
-      if (Diag(index)%id .gt. 0 .or. Diag_coarse(index)%id .gt. 0) then
+      requested = Diag(index)%id .gt. 0 .or. Diag_coarse(index)%id .gt. 0
+      if (requested .and. Diag(index)%diag_manager_controlled) then
```

Comment:
I'm probably not following correctly... but what if `diag_manager_controlled` is not defined? It looked like some of the diagnostics in `GFS_diagnostics` have it set to true, but for others it's not defined. Put another way, where is the default set?

Reply:
I set the default to `.false.` here:
https://github.com/VulcanClimateModeling/fv3gfs-fortran/blob/be96c105bddc89309d2779f807e5184a10c8b5b5/FV3/gfsphysics/GFS_layer/GFS_diagnostics.F90#L41